docs/DefaultDeployment.md

```diff
@@ -57,6 +57,9 @@ We recommend starting with the SQL deployment as it offers the most features. Ho
 Note that when you deploy from our templates, it is set to deploy with $export enabled. As part of the export setup, a storage account is automatically created along with roles. The creation of this storage account and roles requires that you have contributor access at the subscription level. If the person deploying does not have contributor access at the subscription level, the deployment will fail. To deploy successfully in this scenario, you can deploy with export set to false. If you need to use export, it can be configured to true after deployment by someone with contributor access at the subscription level. It takes a few more steps as someone will need to create and link the storage account (it won’t automatically get created) but from there it will work exactly as if you had set it to true at provisioning.
 
 To deploy the backend SQL Server, Azure Web App, and FHIR server code, use the buttons below to deploy through the Azure Portal. If you would like to protect the FHIR API with token authorization, you will need to supply application registration details as described above.
+###### NOTE
+>If you select a SQL database deployment with the Schema Update set to TOOL, you must run the SQL migration tool immediately after the deployment completes successfully.<br>
+The detailed information about this tool is provided [here](SchemaMigrationTool.md).
```

Comment:
Small nit-pick, can you change the second sentence to "Detailed information about this tool is provided here."

Thanks!

Reply:
@evachen96 : We have made the requested changes. Thanks !